content/chainguard/chainguard-enforce/vulnerabilities/vuln-reports-and-attestations.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Vulnerability reports and Attestations in Chainguard Enforce"
+linktitle: "Vulnerability reports and Attestations"
+aliases:
+- /chainguard/chainguard-enforce/chainguard-enforce-kubernetes/vuln-reports-and-attestations/
+- /chainguard/chainguard-enforce/concepts/vuln-reports-and-attestations/
+type: "article"
+description: "An overview of how Chainguard Enforce handles attestations of vulnerability reports"
+date: 2023-03-19T15:56:52-07:00
+lastmod: 2023-03-19T15:56:52-07:00
+draft: false
+tags: ["ENFORCE", "COSIGN", "VULNERABILITIES", "CONCEPTUAL"]
```

Comment:
```suggestion
tags: ["Enforce", "Cosign", "Vulnerability", "Conceptual"]
```

Making these lower case to be consistent with other tags, and I also changed `Vulnerabilities to `Vulnerability` so that both these new pages show up on the tag page together.